### ISSUE TITLE
inconsistent number of newlines after imports

### ISSUE THREAD

**Issue by DanCardin** [tweekmonster/impsort.vim#16]
The following 3 examples are what they look like after sorting. I would basically expect all imports to be separated from the imports by 2 lines (though I realize example 2 might be on purpose), but only the first example does that.

```
import module


def asdf():
    pass
```

```
import module

@decorator
def asdf():
    pass
```

```
import module

value = 4
```

**Comment by tweekmonster** [comment]
The second example with the decorator is actually the mistake.  It's fixed now.

I was going by PEP8 and it only states that [top-level functions and classes are surrounded by two blank lines](https://www.python.org/dev/peps/pep-0008/#blank-lines).

I added the option `g:impsort_lines_after_imports`.  If you set it to `2`, you'll get the spacing that you want.  Before you use that, consider my reason for not making this default:

Based on the sources I've seen, I believe people follow the "two lines after imports" to maintain a consistency with modules that don't have global variables.  Such as:

``` python
import datetime


def example():
    pass
```

I mostly agreed with that sentiment because I like consistency, but there are times when it doesn't look good (to me, at least).  For example, if two spaces are always added after imports:

``` python
import sys

sys.path.insert(0, '/some/path')

import vendor

def example():
    pass
```

would become:

``` python
import sys


sys.path.insert(0, '/some/path')

import vendor


def example():
    pass
```

After working on some large Python 3 projects that required backward compatibility, I changed my opinion about the blank lines that come after a group of imports.  I think that anything in the module-level of the script should remain at the top (with exception to `if __name__ == '__main__':` which should be at the bottom).  Their purpose is to run-once and bootstrap the functions and classes in the module.  Therefore, they should follow the PEP8 rule regarding blank lines in function bodies:

```
Use blank lines in functions, sparingly, to indicate logical sections.
```

IMO, I think that two lines after each set of imports would needlessly space this out too much without improving readability:

``` python
import sys

sys.path.insert(0, '/path/to/site')

import os
import datetime

try:
    from urllib.request import urlopen
except ImportError:
    from urllib2 import urlopen

try:
    import ujson as json
except ImportError:
    import json

global_var = True


class Downloader(object):
    def download(self):
        pass


if __name__ == '__main__':
     Downloader().download()
```
